backend/api/groups.js

```diff
@@ -22,7 +22,7 @@ router.post("/", (req, res) => {
   });
   newGroup
     .save()
-    .then(() => res.json(newGroup))
+    .then((newGroup) => res.json(newGroup))
```

Comment:
Giving the argument the same name as another variable in scope might make things a tiny bit confusing when this code is re-read by another person

Reply:
sorry I misunderstood you. I'll change that right away.